.snippets/code/learn/dapps-list/dapp-directory/urls.json

```diff
@@ -0,0 +1,5 @@
+"urls": {
+    "website": "https://moonbeam.network/",
+    "try": "https://moonbeam.network/",
+    "twitter": "https://twitter.com/MoonbeamNetwork",
```

Comment:
is the comma at the end of twitter extra? either way is fine by me I think should just be consistent.